Sources/TuistCore/Simulator/SimulatorController.swift

```diff
@@ -124,15 +130,20 @@ public final class SimulatorController: SimulatorControlling {
             }
     }
 
+    public func findAvailableDevice(platform: Platform) -> Single<SimulatorDeviceAndRuntime> {
+        self.findAvailableDevice(platform: platform, version: nil, minVersion: nil, deviceName: nil)
```

Comment:
Maybe it might make sense to declare this as `extension` of `SimulatorControlling` protocol? That way you won't have to declare additional method when mocking this.